packaging/cmake/Modules/NetdataUtil.cmake

```diff
@@ -101,9 +101,45 @@ function(netdata_identify_libc _libc_name)
             return()
         endif()
 
+        message(CHECK_FAIL "unknown")
+
+        message(CHECK_START "Looking for libc.so.6")
+        find_program(LIBC_PATH libc.so.6
```

Comment:
On FreeBSD 14 we have `/lib/libc.so.7`. In normal situation we did not reach this part of the code, because `libc` is correctly detected, but considering file will not be available, should we also look for `/lib/libc.so.7`?

Reply:
On FreeBSD this code should never be run, because we don’t build eBPF there at all.